src/Plugin/Visualisation/Source/VisualisationSourceBase.php

```diff
@@ -197,42 +197,42 @@ protected function getIterator() {
   /**
    * {@inheritdoc}
    */
-  public function current() {
+  public function current(): mixed {
```

Comment:
@lem-doghouse, I'm curious about what else could be returned here. Is there a way to avoid mixed return type.

Reply:
Hi @amjad1233 ,

An alternative solution is to add the #[\ReturnTypeWillChange] attribute. It's worth noting that the current() method in the Iterator's Interface is marked with #[TentativeType], indicating that the return type can change dynamically based on runtime conditions.

If you find it suitable, I'm more than willing to make adjustments by replacing the mixed return type and integrating the #[\ReturnTypeWillChange] attribute instead. Please let me know your thoughts on this matter.

Reply:
Oh cool thanks for the run down just checked the interface... Let's leave it as it is for now... I think `#[\ReturnTypeWillChange]` has compatibility issues with older PHP from memory.